src/FileProcessor/TypoScript/Rector/ExtbasePersistenceTypoScriptRector.php

```diff
@@ -116,7 +122,8 @@ public function convert(): ?AddedFileWithContent
             return sprintf('\%s::class', $string);
         });
 
-        return new AddedFileWithContent($this->filename, $content);
+        $fullPath = rtrim($this->configuration['target'], '/') . '/' . ltrim($this->filename, '/');
```

Comment:
It's tedious to check whether a trailing or leading slash is required. So, with this it does not matter what you provide in the config - get the upgrade you're working on done, w/o bothering about slashes.